spec/example_app/app/models/movie_client.rb

```diff
@@ -26,14 +26,49 @@ def top_rated
     end
   end
 
+  def genres
```

Comment:
Do you think this class should be renamed to `TmdbClient` now that it handles genres in addition to movies?

Reply:
Good question. I actually changed it to that at some point, but then for some reason changed it back. I'm gonna change it again. Ch ch ch changes!